Fix TypeScript compilation errors in performance monitor package

## Description
TypeScript compilation is failing due to invalid syntax in the ModuleFederationTracker.

## Error
```
src/ModuleFederationTracker.ts(15,40): error TS1005: '(' expected.
```

## Root Cause
Line 15 contains invalid TypeScript syntax:
```typescript
private originalImport: typeof import;
```

## Fix Required
- Remove the `originalImport` property or properly type it
- `typeof import` is not valid TypeScript syntax

## Files Affected
- `packages/performance-monitor/src/ModuleFederationTracker.ts`

## Priority
High - Blocks all builds and development

## Acceptance Criteria
- [ ] TypeScript compilation succeeds
- [ ] `pnpm build` works without errors
- [ ] No breaking changes to existing functionality

--- a/packages/performance-monitor/src/ModuleFederationTracker.ts
+++ b/packages/performance-monitor/src/ModuleFederationTracker.ts
@@ -12,7 +12,6 @@ export class ModuleFederationTracker {
   private monitor: PerformanceMonitor;
   private config: ModuleFederationConfig;
   private originalFetch: typeof fetch;
-  private originalImport: typeof import;
   private loadCache = new Set<string>();
   private pendingLoads = new Map<string, number>();
 
@@ -191,14 +190,14 @@ export class ModuleFederationTracker {
                 encodedSize: resource.encodedBodySize?.toString() || '0',
                 decodedSize: resource.decodedBodySize?.toString() || '0',
                 protocol: resource.nextHopProtocol || 'unknown',
-                dns: resource.domainLookupEnd - resource.domainLookupStart,
-                tcp: resource.connectEnd - resource.connectStart,
-                ssl: resource.secureConnectionStart > 0
+                dns: (resource.domainLookupEnd - resource.domainLookupStart).toString(),
+                tcp: (resource.connectEnd - resource.connectStart).toString(),
+                ssl: (resource.secureConnectionStart > 0
                   ? resource.connectEnd - resource.secureConnectionStart
-                  : 0,
-                ttfb: resource.responseStart - resource.requestStart,
-                download: resource.responseEnd - resource.responseStart,
-              } as Record<string, string>,
+                  : 0).toString(),
+                ttfb: (resource.responseStart - resource.requestStart).toString(),
+                download: (resource.responseEnd - resource.responseStart).toString(),
+              },
             });
           }
         }

